src/site/_includes/index-page.njk

```diff
@@ -0,0 +1,50 @@
+---
+layout: default-next
+---
+{% from 'macros/post-card.njk' import postCard with context %}
+
+<div class="landing-page">
+ {# @TODO fix subscribe button #}
+ {# <div class="w-actions">
+    {% include 'partials/subscribe-action.njk' %}
+  </div> #}
+
+  {# Hero starts #}
+  <header class="region bg-mid-bg">
+    <div class="wrapper">
+      <div class="headline all-center flow">
+        <h2 class="headline__title">{{ title }}</h2>
+        <p>{{ description }}</p>
+      </div>
+    </div>
+  </header>
+  {# Hero ends #}
+
+  <div class="region">
+    {# Elements start #}
+    <div class="wrapper auto-grid">
+      {% for element in paged.elements %}
+        {% set data = element.data %}
+
+        {{ postCard(
+          element.overrideTitle or element.title | i18n(locale),
```

Comment:
This is too many params to pass into a function. If you have > 3 params, it is better to pass in object instead, e.g.:

{% set post = {
   title: element.overrideTitle or element.title | i18n(locale),
   description: element.description | i18n(locale),
   ...
 } %}
{{ postCard(post) }}

This is true not only for macros but for any js functions in general.